website/static/js/site.js

```diff
@@ -166,7 +166,11 @@
      */
     $.osf.urlParams = function(str) {
         return (str || document.location.search).replace(/(^\?)/,'').split('&')
-            .map(function(n){return n = n.split('='),this[n[0]] = decodeURIComponent(n[1]).replace(/\+/g, ' '),this;}.bind({}))[0];
+            .map(function(n){return n = n.split('='),this[n[0]] = decodeURIComponent(n[1].replace(/\+/g, ' ')),this;}.bind({}))[0];
```

Comment:
It was decoding the URI, then replacing `+` with ` `. This made it impossible to include a `+` character as part of the query, because even after it was decoded from its URL-encoded form (`%2B`), it was then replaced with a space.

By doing the replacement prior to decoding, `+`s get replaced with spaces before `%2B` is replaced with `+`.